src/components/Rockets.js

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getRocketsApi, addReserved, removeReserved } from './Redux/Rockets/rocketSlice';
+import '../App.css';
+
+const RocketItem = ({
+  id, name, image, description, isReserved, dispatch,
+}) => (
+  <div key={id} className="eachRocket">
+    <div className="eachRocketImg">
+      <img src={image} alt="rocketImage" />
+    </div>
+    <div className="eachRocketInfo">
+      <h2>{name}</h2>
+      <div>
+        {isReserved && <span className="smallBtnCheck">Reserved</span>}
+        <p>{description}</p>
+      </div>
+      {isReserved ? (
+        <button
+          type="button"
+          className="cancelReservedBtn"
+          onClick={() => {
+            if (isReserved) {
+              dispatch(removeReserved(id));
+            } else {
+              dispatch(addReserved(id));
+            }
+          }}
+        >
+          Cancel Reservation
+        </button>
+      ) : (
+        <button
+          type="button"
+          className="addReservedBtn"
+          onClick={() => {
+            if (isReserved) {
+              dispatch(removeReserved(id));
+            } else {
+              dispatch(addReserved(id));
+            }
+          }}
+        >
+          Reserve Rocket
+        </button>
```

Comment:
- [ ] Please style the "Reserve Rocket" and "Cancel Reservation" button according to the design guideline 🙏 
### Design Guideline
![image](https://github.com/bhushan354/Space-Travelers-Project/assets/46839548/307f8b19-aaf9-44ab-87f5-d7b1e76aa4d4)

### Your implementation
![image](https://github.com/bhushan354/Space-Travelers-Project/assets/46839548/ffd52930-71e9-4b23-b941-2c95d079e1c0)